skimage/restoration/_denoise.py

```diff
@@ -64,6 +68,20 @@ def denoise_bilateral(image, win_size=5, sigma_range=None, sigma_spatial=1,
     >>> noisy = np.clip(noisy, 0, 1)
     >>> denoised = denoise_bilateral(noisy, sigma_range=0.05, sigma_spatial=15)
     """
+    if multichannel:
+        if image.shape[2] not in (3, 4):
+            msg = "denoise_bilateral is expected to work correctly for " \
```

Comment:
How about we change the message to:

``Input image must be grayscale, RGB, or RGBA, but has shape {0}``